gstreamer-sdp/src/s_d_p_connection.rs

```diff
@@ -10,17 +10,18 @@ use std::mem;
 use std::ffi::CStr;
 
 use ffi;
+use glib::translate::*;
 
 use auto::SDPResult;
 
-
-pub struct SDPConnection(ffi::GstSDPConnection);
+#[repr(C)]
+pub struct SDPConnection(pub ffi::GstSDPConnection);
```

Comment:
IMHO these wrapper structs should not make their content `pub`. If anything, make them `pub(crate)` as needed

Reply:
Oh nice! Didn't know about `pub(crate)`.